src/OrchardCore.Modules/OrchardCore.OpenId/Views/OpenIdServerSettings.Edit.cshtml

```diff
@@ -84,7 +82,7 @@
             @T["Enable Token Endpoint"]
         </label>
     </div>
-    <span class="hint">@T["Enables action {0}", OpenIddictServerOptions.TokenEndpointPath]</span>
+    <span class="hint">@T["Enables action {0}", "/connect/token"]</span>
```

Comment:
Why don't we take it from options anymore? Is that because the complexity and bugs are not worth it? Having them in the UI is another thing I saw you mentioned but I am talking about configuration here only. At the same time we don't make other api endpoints configurable so I am totally fine with this choice, just wondering the reason.

Reply:
Now that endpoint paths are stored in `OpenIdServerSettings` (from which `OpenIddictServerOptions` are later populated by `OpenIdServerConfiguration`), we could probably do something like `Model.TokenEndpointPath.Value ?? "/connect/token"` but I'm not sure it's really worth it.

Actually, the more I think about it, the more I'm tempted to remove these hints, as they almost encourage folks to do it wrong by using hardcoded paths instead of relying on the discovery document (always hosted at `/.well-known/openid-configuration`) to resolve the paths dynamically.